src/Modules/Header/HeaderDataComponent.js

```diff
@@ -22,9 +22,9 @@ const HeaderDataComponent = () => {
                         Food Resources Near You
                     </h1>
                 }
-                    {location.pathname != RENDER_URL.EVENT_LIST_URL && <p className="text-center">
+                    {location.pathname !== RENDER_URL.EVENT_LIST_URL && <p className="text-center">
                     We’re here to help! Input your zip code to find food access resources in your
-                    neighborhood.
+                    community.
```

Comment:
Adding this as to fix [issue 74](https://github.com/midohiofoodbank/freshtrak-public/issues/74)